frontend/packages/noobaa-storage-plugin/src/plugin.ts

```diff
@@ -119,8 +119,8 @@ const plugin: Plugin<ConsumedExtensions> = [
       position: GridPosition.RIGHT,
       loader: () =>
         import(
-          './components/data-resiliency-card/data-resiliency-card' /* webpackChunkName: "object-service-data-resiliency-card" */
-        ).then((m) => m.default),
+          './components/activity-card/activity-card' /* webpackChunkName: "object-service-activity-card" */
+        ).then((m) => m.ActivityCard),
```

Comment:
```suggestion
        ).then((m) => m.default),
```

Reply:
Any specific reason or just for convention.
We are not using HOC here thats why I did a named export.

Reply:
Convention also and the using default gives us the flexibility of not change the function name in imports when the function name is changed. So it prevents us from getting into error conditions.